src/component-queries/convert-data.ts

```diff
@@ -46,34 +42,11 @@ export const convertFrontmatterToDiseaseCellLine = (
                 symbol: cellLineNode.frontmatter.parental_line.frontmatter.gene
                     .frontmatter.symbol,
             },
-
         },
         key: cellLineNode.id,
     };
 };
 
-// todo consolidate this with the above unpacking method
-export const unpackEditingDesignData = (editing_design?: {
-    crna_target_site?: string;
-    dna_donor_sequence?: Sequence[];
-    cas9?: string;
-    f_primer?: string;
-    r_primer?: string;
-    diagrams?: Diagram[];
-}): UnpackedEditingDesignData => {
-    if (!editing_design) {
-        return {};
-    }
-    return {
-        crnaTargetSite: editing_design.crna_target_site,
-        dnaDonorSequence: editing_design.dna_donor_sequence,
-        cas9: editing_design.cas9,
-        fPrimer: editing_design.f_primer,
-        rPrimer: editing_design.r_primer,
-        diagrams: editing_design.diagrams,
-    };
-};
-
```

Comment:
this has been moved, unchanged, to the new convert-data page in the subpage folder